docs/docs/outputs/logging.md

```diff
@@ -1,55 +1,90 @@
-Configure log severity:
+# Defining Tracee Logs
+
+This section showcases how to configure the way Tracee collects logs. This is done through the configuration file. For more information, have a look at the respective section in the [installation guide.](../install/index.md)
+
+## Examples
```

Comment:
for me it's weird the doc starts with examples before it explained anything to me. but in this case, it might be just a wrong title since we do explain everything along with the examples. so think just renaming this will help. suggestion: ## Log options